swarm/__init__.py

```diff
@@ -0,0 +1,11 @@
+#  -*- coding: utf-8 -*-
```

Comment:
For my own benfit, what are these things?

Reply:
Hmm, my IDE adds it automatically.